src/js/components/Select/Select.js

```diff
@@ -271,8 +273,13 @@ const Select = forwardRef(
   },
 );
 
-Select.defaultProps = { ...defaultProps };
-
+Select.defaultProps = {
+  closeOnChange: true,
+  dropAlign: { top: 'bottom', left: 'left' },
+  messages: { multiple: 'multiple' },
+  clear: false,
```

Comment:
Can we just have `undefined`? if it is false than doc.js will need to be updated with the false defaultValue as well.

Reply:
clear is of type bool or object. so false is in?